diff --git a/options.cmake b/options.cmake
index 3cb55d18..2728235c 100644
--- a/options.cmake
+++ b/options.cmake
@@ -13,7 +13,7 @@ option(USE_SDL2_IO "Use SDL2 for all File/Stream IO." ON)
 option(AUTO_SWITCH_CONTROLLER "Automatically switches between keyboard/gamepad depending on what is connected." ON)
 option(MISC_PC_MENU_FIXES "Small bug fixes and improvements for AO Load Menu, AO Keyboard Config Menu, and AE Controller Menu." ON)
 option(ORIGINAL_PS1_BEHAVIOR "Fixes bugs in the PSX Emu layer / Gameplay to match PS1 version of the game." ON)
-option(ORIGINAL_GAME_FIXES "Fixes ALL known gameplay bugs" ON)
+option(ORIGINAL_GAME_FIXES "Fixes ALL known gameplay bugs" OFF)
 option(ORIGINAL_GAME_FIX_AUTO_TURN "Fixes the auto-turn bug commonly used in speedruns" OFF)
 option(ORIGINAL_GAME_FIX_DEATH_DELAY_AO "Fixes the death delay glitch commonly used in speedruns" OFF)
 option(RENDERER_OPENGL "Use OpenGL hardware accelerated rendering." OFF)
